flow-typed/npm/express_v4.x.x.js

```diff
@@ -78,7 +78,7 @@ declare type express$SendFileOptions = {
   dotfiles?: 'allow' | 'deny' | 'ignore'
 };
 
-declare class express$Response extends http$ClientRequest mixins express$RequestResponseBase {
+declare class express$Response extends http$ServerResponse mixins express$RequestResponseBase {
```

Comment:
don't know why they extend response from http$ClientRequest, it doesn't have some methods ( i need writeHead() one)